client/src/connection_cache.rs

```diff
@@ -32,7 +32,7 @@ static MAX_CONNECTIONS: usize = 1024;
 
 /// Used to decide whether the TPU and underlying connection cache should use
 /// QUIC connections.
-pub const DEFAULT_TPU_USE_QUIC: bool = false;
+pub const DEFAULT_TPU_USE_QUIC: bool = true;
```

Comment:
With this set to true we should disallow the user from using disable-quic-servers set to turn it off. Otherwise, forwarded txns to the node itself via quic will be broken

Reply:
Thanks!

I'll deprecate the arg.